sample/cf-nodejs-logging-support-sample/app.js

```diff
@@ -89,9 +102,23 @@ app.get("/correlationandtenantid", function (req, res) {
   var correlationId = reqLogger.getCorrelationId();
   var tenantId = reqLogger.getTenantId();
   reqLogger.info("Correlation ID: %s Tenant ID: %s", correlationId, tenantId);
-  res.send();
+  res.send(lastMessage);
 });
 
+app.get("/binding_information", function (_, res) {
+  result = "There are the following bindings: <br>"
+  if (process.env.VCAP_SERVICES)
+    var services = JSON.parse(process.env.VCAP_SERVICES)
+  console.log(services)
+  if (services == undefined)
+    res.send("There are no bindings present for this application")
+  else {
+    for (var service in services)
+      result += service + "<br>"
+    res.send(result)
```

Comment:
Just to confirm: This prints the names of the services only.

Reply:
Yes, this only gets the service names